SpatialGDK/Documentation/index.md

```diff
@@ -17,13 +17,9 @@ SpatialOS provides:<br/>
 
 ## Multiserver networking
 
-Multiserver functionality is available through either:
-
-* server **offloading** (available in alpha), in which Unreal server functionality is split between multiple servers and those servers compute different functionality across the whole game world, or 
-* server **zoning** (available in pre-alpha), in which the game world is split into several geographical areas and each area has a dedicated Unreal server computing all the functionality for it.
-
-<%(Callout  message="We do not currently recommend development using multiserver zoning functionality. For more information on multiserver zoning availability, see the [development roadmap](https://github.com/spatialos/UnrealGDK/projects/1) and [Unreal features support]({{urlRoot}}/unreal-features-support) page.")%>
+Multiserver functionality is currently available through server **offloading** (available in alpha), in which Unreal server functionality is split between multiple servers and those servers compute different functionality across the whole game world.
```

Comment:
I think it's fair to say SpatialOS has two types of zoning, and then downplay the availability of zoning for the GDK. This makes it a bit easier to cast your eyes over?!

```suggestion
SpatialOS has two types of multiserver networking:
* Server **offloading** (available in the GDK in alpha), in which Unreal server functionality is split between multiple servers and those servers compute different functionality across the whole game world.
```